cmd/opentelemetry/app/defaultconfig/default_config_test.go

```diff
@@ -59,6 +63,41 @@ func TestService(t *testing.T) {
 				},
 			},
 		},
+		{
+			cfg: ComponentSettings{
+				ComponentType: Collector,
+				StorageType:   "badger",
+			},
+			service: configmodels.Service{
+				Extensions: []string{"health_check"},
+				Pipelines: configmodels.Pipelines{
+					"traces": &configmodels.Pipeline{
+						InputType:  configmodels.TracesDataType,
+						Receivers:  []string{"otlp", "jaeger"},
+						Processors: []string{"batch"},
+						Exporters:  []string{"jaeger_badger"},
+					},
+				},
+			},
+		},
+		{
+			cfg: ComponentSettings{
+				ComponentType: Agent,
+			},
+			service: configmodels.Service{
+				Extensions: []string{"health_check"},
+				Pipelines: configmodels.Pipelines{
+					"traces": &configmodels.Pipeline{
+						Name:       "traces",
+						InputType:  configmodels.TracesDataType,
+						Receivers:  []string{"otlp", "jaeger"},
+						Processors: []string{"batch", "queued_retry"},
+						Exporters:  []string{"jaeger"},
+					},
+				},
+			},
+			otelConfig: "testdata/addqueuedprocessor.yaml",
```

Comment:
This file does not exist

Reply:
whoops.  forgot to add it.  just pushed.